src/Package/Impl/Packages/BasePackage.cs

```diff
@@ -57,20 +58,22 @@ public abstract class BasePackage<TLanguageService> : VisualStudio.Shell.Package
                 RegisterEditorFactory(editorFactory);
             }
 
-            var menuCommandService = (IMenuCommandService)GetService(typeof(IMenuCommandService));
-            foreach (var commmand in CreateMenuCommands()) {
-                menuCommandService.AddCommand(commmand);
-            }
+            IdleTimeAction.Create(() => {
```

Comment:
B/c of DTE.MainWindow access from commands (via interactive workflow -> connection manager)